docs-ui/stories/components/replays/replayPage.stories.js

```diff
@@ -13,10 +17,38 @@ export default {
   component: ReplayPlayer,
 };
 
+const PanelHeader = styled(_PanelHeader)`
+  display: block; /* Override flex */
+  padding: 0; /* The disablePadding prop doesn't disable all the padding */
+`;
+
+const ManualResize = styled('div')`
+  resize: vertical;
+  overflow: auto;
+  max-width: 100%;
+
+  ${p =>
+    p.isFullscreen
+      ? `resize: none;
+      width: auto !important;
+      height: auto !important;
+      `
+      : ''}
+`;
+
 export const PlayerWithController = () => (
   <ReplayContextProvider events={rrwebEvents1}>
-    <ReplayPlayer />
-    <ReplayController speedOptions={[0.5, 1, 2, 8]} />
+    <Panel>
+      <PanelHeader>
+        <ManualResize isFullscreen={false}>
+          <ReplayPlayer />
+        </ManualResize>
+      </PanelHeader>
+      <Scrubber />
+      <PanelBody withPadding>
+        <ReplayController toggleFullscreen={action('toggleFullscreen')} />
+      </PanelBody>
+    </Panel>
```

Comment:
Would it be worthwhile to refactor our details components so that we're not duplicating code and trying to keep them synced?

Reply:
yeah, i'll send this back to draft and take care of that in a bit.